Makefile

```diff
@@ -18,4 +20,11 @@ codeqlbuild:
 
 # Rule to setup the project. This is a special case because it's not a Mage target.
 setup:
-	@go run magefiles/setup/setup.go
\ No newline at end of file
+	@go run magefiles/setup/setup.go
+
+BINDIR ?= $(GOPATH)/bin
+CURRENT_DIR = $(shell pwd)
+
+test-sim-after-import:
```

Comment:
@calbera I'm down to start migrating away from mage tbh, more hassle than its been a value add.